sources/packages/backend/libs/sims-db/src/entities/disbursement-values.model.ts

```diff
@@ -100,4 +101,35 @@ export class DisbursementValue extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   disbursementSchedule: DisbursementSchedule;
+  /**
+   * Amount that is subtracted because of a restriction.
+   */
+  @Column({
+    name: "restriction_amount_subtracted",
+    type: "numeric",
+    nullable: true,
+    transformer: numericTransformer,
+  })
+  restrictionAmountSubtracted?: number;
+  /**
+   * References the restriction related to the disbursement
+   * due to which the amount for subtracted.
+   */
+  @RelationId(
+    (disbursementValue: DisbursementValue) =>
+      disbursementValue.restrictionSubtracted,
+  )
+  restrictionIdSubtracted?: number;
+  /**
+   * Restriction id that was placed for the student due to which the award amount was reduced.
```

Comment:
Please see the same comment for the DB column value. If it is updated there please ensure to change this one also.